pkg/vars/vars.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vars
+
+import (
+	"time"
+
+	"github.com/pingcap/tiflow/cdc/model"
```

Comment:
It looks so weired for one package in the `pkg` folder, reference some other package in the `cdc`

Reply:
> It looks so weird for one package in the `pkg` folder, reference some other package in the `cdc`

Agreed!

Reply:
moved to cdc package